config/m4/compiler.m4

```diff
@@ -143,7 +143,7 @@ case $CC in
                 -Wno-free-nonheap-object "
         CXXFLAGS="$CXXFLAGS -Wshadow -Wall -Wextra -Werror -Wundef \
                 -ffunction-sections -fdata-sections -Wsequence-point -pipe -Winit-self -Wmissing-include-dirs \
-                -Wno-free-nonheap-object "
+                -Wno-free-nonheap-object -Wno-overloaded-virtual "
```

Comment:
Is it really OK to avoid this warning? Wouldn't it be better to fix it?

Reply:
I did not find a way to fix them in simple manner.
Task for fixing this and others option can be planned.